packages/app-store/routing-forms/pages/reporting/[...appPages].tsx

```diff
@@ -60,25 +69,53 @@ const Result = ({ formId, jsonLogicQuery }: { formId: string; jsonLogicQuery: Js
     return <div>Error loading report {error?.message} </div>;
   }
   headers.current = (data?.pages && data?.pages[0]?.headers) || headers.current;
+
   const numberOfRows = data?.pages.reduce((total, page) => total + (page.responses?.length || 0), 0);
+  const downloadButtonDisabled = !numberOfRows || !data || !headers.current;
+  const handleDownload = () => {
+    if (downloadButtonDisabled) {
+      return;
+    }
+
+    const header = `${headers.current ?? [].join(",")}\n`;
+    const rows = data.pages.flatMap((page) =>
+      page.responses.map(
+        (response) => `${response.map((value) => (value.includes(",") ? `"${value}"` : value)).join(",")}\n`
```

Comment:
@CarinaWolli this is the fix

Reply:
This is how it will look like:

<img width="263" alt="Screenshot 2024-10-31 at 14 28 07" src="https://github.com/user-attachments/assets/3f64bf4b-4c0b-4bd8-84c5-805990aaeb53">